modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/rest/api/SCIMUtils.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.scim2.rest.api;
+
+import io.restassured.response.Response;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpHeaders;
+import org.testng.Assert;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+
+/**
+ * Common utils for SCIM related test cases.
+ */
+public class SCIMUtils {
+
+    private static final String SCIM_USER_SCHEMAS = "[urn:ietf:params:scim:schemas:core:2.0:User, " +
+            "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User]";
+
+    public static void validateSchemasAttribute(Object schemasAttribute) {
```

Comment:
Shall we add method level comment to each public method?

Reply:
Resolved in https://github.com/wso2/product-is/pull/7362/commits/1aa6b856847559ff0eaa3e4c3f520ab86ea8c16f